### PR TITLE
Fix parsing solver name for Python 3.10

### PR DESCRIPTION
**Pull request by fridex**
## Related Issues and Dependencies

Related: https://github.com/thoth-station/s2i-thoth/issues/231#issuecomment-1029030336

## This introduces a breaking change

- [x] No

### CODE REVIEW

File: thoth/common/openshift.py [hunk 1]
```diff
@@ -217,7 +217,7 @@ def parse_python_solver_name(cls, solver_name: str) -> Dict[str, Any]:
                 f"Python version encoded into Python solver name does not start with 'py' prefix: {solver_name!r}"
             )
 
-        python_version = ".".join(list(python_version))
+        python_version = python_version[0] + "." + python_version[1:]
```
Comment (KPostOffice) [inline thoth/common/openshift.py:220]: Can you use f-string instead of concatenation?

Comment (fridex) [reply thoth/common/openshift.py]: Adjusted, PTAL 👍🏻

**Review by harshad16 — APPROVED** [review]
lgtm
thanks :100:

**Comment by fridex** [comment]
> /approve

/approve

**Pull request merged**